resources/leiningen/new/kraken_http_api/server.clj

```diff
@@ -19,5 +19,5 @@
 
 (defn -main [& args]
   (let [rabbit-resources (queue/initialize)]
-    (start-http-server (config [:server]))
+    (start-http-server)
```

Comment:
I was using user-http-api as my guide and it doesn't pass that arg. All the rest do, though. I'll revert that.